docusaurus.config.js

```diff
@@ -80,6 +80,18 @@ const config = {
         'Licensed by ComCODE with <a href="https://creativecommons.org/licenses/by-nc-sa/4.0/" target="_blank" rel="noopener noreferrer">CC BY-NC-SA 4.0</a> for creative and educational use.',
     },
   },
+
+  // Global markdown config
+  // See: https://docusaurus.io/docs/api/docusaurus-config#markdown
+  markdown: {
```

Comment:
How did you pick these settings?

Reply:
The MDX1 compatibility settings were chosen based off the Docusaurus recommendations, listed in the PR description. I explicitly set `headingIds` value to its default to avoid a potential regression issue in case the default behaviour changes in the future.

As I was adding the markdown config anyway, I explicitly set `format` and `mermaid` to their default values also.